internal/services/v1/permissions.go

```diff
@@ -393,158 +393,7 @@ func TranslateExpansionTree(node *core.RelationTupleTreeNode) *v1.PermissionRela
 	}
 }
 
-const lrv2CursorFlag = "lrv2"
-
 func (ps *permissionServer) LookupResources(req *v1.LookupResourcesRequest, resp v1.PermissionsService_LookupResourcesServer) error {
-	// If the cursor specifies that this is a LookupResources2 request, then that implementation must
-	// be used.
-	if req.OptionalCursor != nil {
-		_, ok, err := cursor.GetCursorFlag(req.OptionalCursor, lrv2CursorFlag)
-		if err != nil {
-			return ps.rewriteError(resp.Context(), err)
-		}
-
-		if ok {
-			return ps.lookupResources2(req, resp)
-		}
-	}
```

Comment:
I figured this was no longer necessary since the flag itself isn't really necessary anymore.